src/test/java/org/jenkins-ci/tools/Junit4DummyTest.java

```diff
@@ -0,0 +1,14 @@
+package org.jvnet.maven.jellydoc;
+
+import org.junit.Test;
+
+public class Junit4DummyTest {
```

Comment:
@NotMyFault since [the run](https://ci.jenkins.io/blue/organizations/jenkins/Tools%2Fgradle-jpi-plugin/detail/master/1/tests) did register all the existing tests, I'm going to remove this one. No action required, just wanted to let you know so it wasn't surprising in the future.